index.js

```diff
@@ -3,51 +3,18 @@
 module.exports = {
   name: require('./package').name,
 
-  included: function(app) {
+  included(app) {
     this._super.included.apply(this, arguments);
 
-    this._welcomeConfig = app.options['ember-welcome-page'] || {};
-
-    if (this._isDisabled()) { return; }
-
     app.import('vendor/welcome-page.css');
   },
 
-  config: function() {
-
+  config() {
```

Comment:
We should remove this also, any MU support code should be released before the next major.

Reply:
I had initially removed all MU code, but @locks wanted me to remove those changes to make the changelog cleaner and do it in another PR.